### PR TITLE
Android

### LINKED ISSUE #38: Android APK on Site
As requested here:
https://help.nextcloud.com/t/about-downloading-apk-file-of-android-app-from-site/668
We can provide an APK file.
Where and how should we do this?

### PR DESCRIPTION
**Pull request by jospoortvliet**
fixes #38 and makes the [folks here happy](https://help.nextcloud.com/t/about-downloading-apk-file-of-android-app-from-site/668/9) :smile_cat:

### CODE REVIEW

**Comment by jospoortvliet** [comment]
screenshot...
install - nextcloud 5

**Comment by mar1u5** [comment]
What about resizing the F-Droid and Google Play button, so that we can show them in one row 😁

**Comment by AndyScherzinger** [comment]
Alternatively you can also use this link ;) https://f-droid.org/repository/browse/?fdfilter=com.nextcloud

File: page-install.php [hunk 1]
```diff
@@ -28,8 +28,11 @@
 		<p>Connect to your Nextcloud with our Android app<!--clients for your devices-->:</p>
 		<a style="opacity: 0.9" target="_blank" href="<?php echo $DOWNLOAD_CLIENT_MOBILE_ANDROID; ?>">
 		<img src="<?php echo get_template_directory_uri(); ?>/assets/img/clients/buttons/googleplay.png">
+		</a> 
+		<a style="opacity: 0.9" target="_blank" href="<?php echo $DOWNLOAD_CLIENT_MOBILE_FDROID; ?>">
+		<img src="<?php echo get_template_directory_uri(); ?>/assets/img/clients/buttons/fdroid.png">
 		</a>
-		<p><a href="https://github.com/nextcloud/android">Sources here</a>, more coming! Any WebDAV client or the <a href="https://owncloud.org/install/#install-clients">old ownCloud clients</a> will work with Nextcloud.</p>
+		<p><a href="https://github.com/nextcloud/android">Sources here</a>, <a href="https://download.nextcloud.com/android">APK here</a> and you can help test our upcoming beta <a target="_blank" href="<?php echo $DOWNLOAD_CLIENT_MOBILE_FDROID_BETA; ?>">on F-Droid here</a>. Any WebDAV client or the <a href="https://owncloud.org/install/#install-clients">old ownCloud clients</a> will work with Nextcloud.</p>
```
Comment (LukasReschke) [inline page-install.php]: If we link to the beta we should likely also mention https://play.google.com/apps/testing/com.nextcloud.client but this gets kinda crowded 🙈

**Comment by jospoortvliet** [comment]
install - nextcloud 7
install - nextcloud 6

So how about that... ;-)

**Comment by jospoortvliet** [comment]
(this page will need a make-over again soon, when more clients join and all that, so I'd like to get this live rather than trying to perfect it now - if ya'll don't mind...)

**Comment by AndyScherzinger** [comment]
Looking good :+1:

**Comment by mar1u5** [comment]
:+1:

**Comment by jospoortvliet** [comment]
awesome, thanks!

**Pull request merged**